include/RAJA/util/PluginStrategy.hpp

```diff
@@ -25,6 +25,8 @@ class PluginStrategy
     virtual void preLaunch(PluginContext p) = 0;
 
     virtual void postLaunch(PluginContext p) = 0;
+
+    virtual void init(PluginOptions p) = 0;
```

Comment:
I think this function should not be a pure virtual. If you provide an empty implementation here, plugins that don't need to do any extra init don't need to implement it.